app/models/order.rb

```diff
@@ -13,22 +13,23 @@ class Order < ActiveRecord::Base
 
   delegate :name, to: :order_status, prefix: true
 
-  scope :pending, -> { where(order_status_id: 1) }
+  scope :pending, -> { where(order_status_id: OrderStatus.find_by(name: "Pending")) }
```

Comment:
Is `OrderStatus` just a lookup table? It seems like you could cache all of these statuses once at some level in the application and store them in a hash for easy lookup. I assume that ActiveRecord is pretty good at caching, so maybe this is fine.